resources/js/Pages/Products/ProductEdit.vue

```diff
@@ -143,6 +148,14 @@ export default {
             }
         },
 
+        deleteImg(image) {
+            axios.delete(route('api.products.images.delete', [this.model.id, image.id]))
```

Comment:
Can write this shorter as:
axios.delete(route('api.products.images.delete', [this.model, image]))